mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1200,8 +1208,15 @@ func expandCloneContext(configured []interface{}) (*sqladmin.CloneContext, strin
 
 	_cloneConfiguration := configured[0].(map[string]interface{})
 
+	databaseNames := []string{}
```

Comment:
Do we need to read this field from the API? If so, it should be present in the flatten function for this field also.

Reply:
I don't think so. This field should follow the same pattern as `point_in_time` field, which is not read from the API.